resources/sdk/purecloudjavascript/templates/ApiClient.mustache

```diff
@@ -204,6 +206,33 @@ class ApiClient {
 		this.config.setEnvironment(environment);
 	}
 
+    setHttpClient(httpClient) {
+        if (!(httpClient instanceof HttpClient)) {
+            throw new Error("httpclient must be an instance of Abstract HttpClient. See DefualtHttpInstance for a prototype");
+        }
+        this.httpClient = httpClient;
+    }
+
+
+	setHttpAgentPaths(certPath, keyPath, caPath) {
```

Comment:
The setHttpAgentPaths would need to have a switch/test on window === undefined, to have code for node (with require on fs) and code for browser.

Reply:
done